cmd/addon-operator-manager/main.go

```diff
@@ -167,6 +224,8 @@ func main() {
 		initPprof(mgr, opts.pprofAddr)
 	}
 
+	initMetricsRelayServer(mgr, opts.metricsHttpsRelayAddr, opts.metricsAddr)
```

Comment:
Can we make it optional to enable the HTTPS relay server? It could be useful in dev envs where we don't actually care about TLS for /metrics.

Reply:
Done, 
depending on the `--enable-metrics-tls-termination` flag, either an http or an https endpoint will be exposed at `opts.metricsAddr` ("8080")